src/Components/Authorization/AuthForm.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react"
+
+import ForgotPasswordForm from "./ForgotPasswordForm"
+import LoginForm from "./LoginForm"
+import RegisterForm from "./RegisterForm"
+import { FormComponentType, InputErrors, InputValues, Mode } from "./Types"
+
+interface Props extends React.HTMLProps<HTMLFormElement> {
+  mode: Mode
+  values?: InputValues
+  errors?: InputErrors
+  handleSubmit: () => void
+  // signupIntent?: string
+  // destination?: string
+}
+
+interface State extends React.HTMLProps<HTMLFormElement> {
+  mode?: Mode
+}
+
+class AuthForm extends React.Component<Props, State> {
+  static defaultProps: Partial<Props> = {
+    errors: {},
+    values: {},
+    handleSubmit: () => alert("submitting!"),
```

Comment:
for now, this is just to give some feedback when testing the form in storybook. I guess I could have put it in storybook.